### DISCUSSION TITLE
[Question][Hw3] About ocaml language

### DISCUSSION THREAD

**Discussion by ee12ha0220** [prosyslab-classroom/cs348-information-security#516] [Q&A]
1. How do I use utils.ml? If I use the instruction in the tutorial, an error occurs. 
![image](https://user-images.githubusercontent.com/71373857/116352726-949ac680-a830-11eb-8f39-191b91aa217e.png)

2. What is the function of dune file?

**Comment by RiceBiscuits** [comment]
1. You can use utils.ml as a library when you write your code. 

2. Dune will build and test your code.  
If you want to know more details about Dune, please see links below.
(Dune build: https://dune.readthedocs.io/en/stable/quick-start.html#building-a-hello-world-program)
(Dune test: https://dune.readthedocs.io/en/stable/quick-start.html#defining-tests)

**Comment by ee12ha0220** [comment]
Thank you! :)
